lighthouse-core/audits/largest-contentful-paint-element.js

```diff
@@ -33,7 +33,7 @@ class LargestContentfulPaintElement extends Audit {
       title: str_(UIStrings.title),
       description: str_(UIStrings.description),
       scoreDisplayMode: Audit.SCORING_MODES.INFORMATIVE,
-      requiredArtifacts: ['TraceElements'],
+      requiredArtifacts: ['traces', 'TraceElements'],
```

Comment:
add this for filtering audits? Should definitely add a comment explaining that in that case. We probably need to expand our dependency tracking for that

Reply:
yah for use with ` --only-audits=layout-shift-elements`